Nest/00. Plan.md

```diff
@@ -0,0 +1,67 @@
+# 1 Introduction
+## 1.1 Package manager
+
+## 1.2 Packages and dependencies
+
+## 1.3 Repositories
+
+## 1.4 Package's syntax
+
+# 2 Nest
+## 2.1 Nest's options
+
+## 2.2 Nest commands
+### 2.2.1 Synchronizing the local cache
+### 2.2.2 Troubleshooting
+
+## 2.3 Installing and uninstalling a package
+### 2.3.1 Installing a package
+### 2.3.2 Uninstalling a package
+### 2.3.3 Troubleshooting
+
+## 2.4 Updating your system
```

Comment:
System may be unclear or vague."operating system" or even "OS" (with a description when then abbreviation is introduced) would be more precise.

Reply:
(also the "your" could be replaced by "the")

Reply:
I actually disagree with you on this. I think the system is the least-worst word to describe the whole thing.

I fear "Updating the Operating System" has too much of a low level connotation and may be mixed up with the idea of "Updating the kernel", which may not reflect the idea of updating user software (with GUI).

Reply:
Agreed about the confusion concerns. Better to keep as is.

Reply:
:+1: I'll put it back as 'system'